src/release-specification.ts

```diff
@@ -317,6 +343,63 @@ export async function validateReleaseSpecification(
         }
       }
 
+      // Check to compel users to release packages with breaking changes alongside their dependents
+      if (
+        versionSpecifierOrDirective === 'major' ||
+        (isValidSemver(versionSpecifierOrDirective) &&
+          diff(pkg.validatedManifest.version, versionSpecifierOrDirective) ===
```

Comment:
`pkg` and `packageName` are also kind of confusing here, but there's nothing we can do about that right now. Perhaps we keep this in mind for a future refactor, though.

Reply:
I did update pkg to changedPackage and packageName to changedPackageName